config/pipeline.yaml

```diff
@@ -318,6 +318,29 @@ jobs:
       dtbs_check: true
     kcidb_test_suite: dtbs_check
 
+  kbuild-gcc-12-arm64-preempt_rt:
+    <<: *kbuild-gcc-12-arm64-job
+    params:
+      <<: *kbuild-gcc-12-arm64-params
+      fragments:
+       - 'preempt_rt'
+      defconfig: defconfig
+    rules:
+      tree:
+      - 'stable-rt'
+
+  kbuild-gcc-12-arm64-preempt_rt_chromebook:
```

Comment:
nit: section separator is `-` instead of `_` (unless `_`s are used in fragment/defconfig name) but that's something to keep an eye on in the future, no need to amend this patchset

Reply:
Thanks!